t/t1017-update-db.t

```diff
@@ -87,4 +96,70 @@ test_expect_success 'get all the columns from the queue_table and make sure the
 	test_cmp queue_table_columns.expected queue_table_columns.test
 '
 
+test_expect_success 'successfully update flux-accounting DB from v0.10.0' '
```

Comment:
You could make use of a for loop here to avoid using a variable per db version. Then, all you would need to do is drop a new database into `expected/test_dbs` for it to be checked in this test file.

For an example, see the end of flux-core `t/t2219-job-manager-restart.t`:

https://github.com/flux-framework/flux-core/blob/3c367a3a15e888857cab2e8dbf2704e97861a494/t/t2219-job-manager-restart.t#L57-L66

Reply:
Ah, nice, thanks for this suggestion. If it's okay with you, I'd like to open an issue on this and work on this in the near future, just for the purposes of landing this PR by today.